.gitignore

```diff
@@ -12,5 +12,7 @@ CHANGELOG.md
 *.bisign
 *.tar.gz
 *.zip
+hemtt*
```

Comment:
Just use ./hemtt

Reply:
You were the one to add this, but I'll change it.

Reply:
Whoops, then I guess my mind changed :D

@jonpas does the template folder need to be gitignored or is it safe to include in git?

Reply:
It's your template, entirely depends where you want to host it. However, considering it's not even fully implemented in current HEMTT, it doesn't matter either way.

Reply:
Then I guess, for now, we can gitignore it.